### PR TITLE
:memo: Add setup in readme

### PR DESCRIPTION
**Pull request by tbetous**
add setup in readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,38 +45,149 @@ Want to setup this stack ?
...
+### Tips
+
+We highly recommand to use [VSCode](https://code.visualstudio.com/) with the following plugins to get a better development experience :
+
+- Prettier
+- Eslint
+- Vetur
+
+**bento-starter** comes with a default code editor config that will automatically be used by vscode. This config is available in `.vscode/settings.json`.
+:::
```
Comment (kefranabg) [inline README.md]: You can remove this

File: README.md [hunk 2]
```diff
@@ -45,38 +45,149 @@ Want to setup this stack ?
...
+npm i
+```
+
+### Step 2 - Firebase configuration
+
+🕙 Estimated time → **3 minutes**
+<br />
+
+- Create a new firebase project with the [firebase console](https://console.firebase.google.com)
+- Once your firebase project is created, add an application by clicking the web button 👉 ![Firebase web app button](/assets/img/firebase-web-btn.jpg) and copy the config object and replace the config variable in `/src/firebase/init.js` in bento-starter project.
```
Comment (kefranabg) [inline README.md]: You have to add the `firebase-web-btn.jpg`in the `resources` folder of this repos (Currently getting a 404)

File: README.md [hunk 3]
```diff
@@ -45,38 +45,149 @@ Want to setup this stack ?
...
+- Go to [https://circleci.com](https://circleci.com)
+- Login with your github account
+- Authorize CircleCI to look into your github projects
+- Go to `Side menu → Add projects` and click the `Set Up Project` button corresponding to your **bento-starter** project
+- Choose `Linux` for operating system and `Node` for the language
+- You can directly start your first CircleCI build by clicking `Start building` button.
+- Go to `Side menu → Jobs` and you should see your first CircleCI job running
+- Now wait for all the jobs to finish
+
+Your last build (`deploy`) will fail and this is normal :sweat_smile: It's because of the deployment step (`npm run firebase:deploy:ci`). We need to authorize circle ci to deploy on our firebase hosting project. For this we just need to add a firebase token to circle ci :
```
Comment (kefranabg) [inline README.md]:
```suggestion
The last job (`deploy`) 
```

Comment (kefranabg) [reply README.md]: I made change in the bento-starter-docs repos

File: README.md (hunk 2 above)
Comment (tbetous) [reply README.md]: Nice !

**Pull request merged**